types/post_proof_partitions.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// PoStProofPartitions TODO(laser)
 type PoStProofPartitions uint64
 
 const (
+	// TestPoStPartitions TODO(laser)
 	TestPoStPartitions = PoStProofPartitions(iota)
+	// OnePoStPartition TODO(laser)
```

Comment:
OnePoStPartition indicates that a single partition was used to create a PoSt proof.